cmsplugin_filer_image/templates/cmsplugin_filer_image/plugins/image/default.html

```diff
@@ -9,10 +9,10 @@
 	{% else %}
 		{% thumbnail instance.image opts.size crop=opts.crop upscale=opts.upscale as thumbnail %}
 	{% endif %}
-	{% if link %}<a href="{{ link }}"{% if instance.target_blank %} target="_blank"{% endif %}>{% endif %}<img{% if instance.alignment %} class="{{ instance.alignment }}"{% endif %} alt="{% if instance.alt %}{{ instance.alt }}{% endif %}" src="{{ thumbnail.url }}"{% if instance.caption %} title="{{ instance.caption }}"{% endif %} />{% if link %}</a>{% endif %}
+	{% if link %}<a href="{{ link }}"{% if instance.target_blank %} target="_blank"{% endif %}{% with attr=instance.attributes_str %}{% if attr %} {{ attr }}{% endif %}{% endwith %}>{% endif %}<img{% if instance.alignment %} class="{{ instance.alignment }}"{% endif %} alt="{% if instance.alt %}{{ instance.alt }}{% endif %}" src="{{ thumbnail.url }}"{% if instance.caption %} title="{{ instance.caption }}"{% endif %} />{% if link %}</a>{% endif %}
 {% endcomment %}
 
-{% if link %}<a href="{{ link }}"{% if instance.target_blank %} target="_blank"{% endif %} class="filer_image_link">{% endif %}
+{% if link %}<a href="{{ link }}"{% if instance.target_blank %} target="_blank"{% endif %} class="filer_image_link"{% with attr=instance.attributes_str %}{% if attr %} {{ attr }}{% endif %}{% endwith %}>{% endif %}
```

Comment:
I would not worry about the possible trailing space and just use `` {{ instance.attributes_str }}``.